src/plugins/Tus10.js

```diff
@@ -46,12 +40,18 @@ export default class Tus10 extends Plugin {
           percentage = Math.round(percentage)
 
           // Dispatch progress event
-          this.core.emitter.emit('progress', {
-            plugin: this,
-            percentage: percentage
+          this.core.emitter.emit('upload-progress', {
```

Comment:
Right, shorthand would be dandy, but yes, this should preferably stay consistent throughout the application. So keeping `this.core.emitter.emit` for now. If we used simple functions, I’d [pass `emitter` as argument](https://github.com/substack/training/blob/3041b1e4e3908d4df1b26cf578c34cd4df8fe9b7/web-dev-whirlwind/scratch/bus/actions.js).